adds simple CSS for inline links in vf-box__text

sturobson opened the pull request:
As we could be swapping the WP [fact box] for `vf-box` we need to incorporate the need for links in the `vf-box__text`. 

This does this by making the `<a>` inherit the `currentColor` from the `<p class="vf-box__text">`.

![Screenshot 2020-03-20 at 10 37 18](https://user-images.githubusercontent.com/925197/77156284-fcf86f80-6a96-11ea-9a3f-4b03177cd5d1.png)

sturobson committed:
adds simple CSS for inline links in vf-box__text

khawkins98 commented:
As this is pretty WP specific, should it instead be some local CSS for vf-wp?

khawkins98 commented:
![image](https://user-images.githubusercontent.com/928100/77156920-ab54e280-6aa0-11ea-8fa8-62b33088c263.png)

WP will probably also need some spacing tweaks on that heading too

sturobson commented:
I think we can incorporate it in `vf-core` as this does then allow factoids or other 'text in `vf-box`' to have a link - as I'm sure that will definitely be a requirement at some point.

sturobson commented:
> WP will probably also need some spacing tweaks on that heading too 

I'm going to go out on a limb and say that _that_ is `vf-wp` specific.

khawkins98 commented:
> I think we can incorporate it in vf-core as this does then allow factoids or other 'text in vf-box' to have a link - as I'm sure that will definitely be a requirement at some point.

In this case it might be more future proof if we do:

```css

.vf-box.vf-content a, 
and/or .vf-content .vf-box a {}
```

sturobson commented:
Yes! It's probably a good idea to add `.vf-content .vf-box a {}`.

I wouldn't expect people to include `vf-content` inside a `vf-box` (unless they're hand coding things (which, they probably shouldn't be doing _really_) … so perhaps I should mention that in the README.md

khawkins98 commented:
👍

sturobson committed:
adds option for vf-box to be child of vf-content and docs

sturobson commented:
I've updated the CSS so that a `vf-box` will now work as a child component insdie `vf-content`. I've also added some simple docs to this effect. I've also gone back and reinstated `text-decoration: underline` so it matches the rest of the system.
